### DISCUSSION TITLE
Ignore "unavailable"

### DISCUSSION THREAD

**Discussion by Eckart0** [dylandoamaral/uptime-card#130] [Improvements] [answered]
Hello,

I have a suggestion for a small extension. Is it possible to add an option like "hide_unavailable" or "ignore_unavailable"?

I also use uptime-card for WiFi sensors with an unstable connection, such as for controlling garden irrigation.
Whenever the sensor switches between "off" and "unavailable", uptime-card interprets this as a brief "on" state and then displays it incorrectly. 

Regards,
Eckart

**Comment by dylandoamaral** [comment, marked as answer]
You can just specify the OK state, any other states should be considered as "off"

**Reply by Eckart0** [reply]
Thx. Easy solution, works. Eckart
